packages/pluggableWidgets/switch-native/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "switch-native",
   "widgetName": "Switch",
-  "version": "1.1.0",
+  "version": "1.0.0",
```

Comment:
This probably went wrong during rebase. It shouldn't go down

Reply:
We will set the version to 1.0.0 and create new separate PR for the changelog